.github/workflows/dev_on_workflow_tester.yml

```
@@ -109,6 +131,85 @@ jobs:
             v1-coverage-dependency-cache-${{ runner.os }}-${{ github.sha }}-${{ matrix.shard }}
           key: v1-coverage-dependency-cache-${{ runner.os }}-${{ github.sha }}-${{ matrix.shard }}
 
+      - name: Cache the allure reports
+        if: github.ref == 'refs/heads/master'
+        id: allure-report-cache-native
+        uses: pass-culture-github-actions/gcs-cache@v1.0.0
+        with:
+          bucket: ${{ inputs.CACHE_BUCKET_NAME }}
+          path: |
+            allure-results-${{matrix.shard}}
+          restore-keys: |
+            v1-allure-dependency-cache-${{ runner.os }}-${{ github.sha }}-${{ matrix.shard }}
+          key: v1-allure-dependency-cache-${{ runner.os }}-${{ github.sha }}-${{ matrix.shard }}
+
+  allure-report:
+    name: Update Allure report
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write
+      contents: read
+    needs: yarn_test_native
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: 'Authentification to Google'
+        uses: 'google-github-actions/auth@v2'
+        with:
+          workload_identity_provider: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
+
+      - name: Set up Cloud SDK to get gsutils
+        uses: 'google-github-actions/setup-gcloud@v2'
+        with:
+          version: '>= 416.0.0'
+
+      - name: 'Retrieve reports from bucket'
+        run: |
+          mkdir allure-results
+          gsutil cp 'gs://${{ inputs.CACHE_BUCKET_NAME }}/pass-culture/pass-culture-app-native/v1-allure-dependency-cache-${{ runner.os }}-${{ github.sha }}-*' allure-results
+          gsutil cp 'gs://${{ inputs.CACHE_BUCKET_NAME }}/pass-culture/pass-culture-app-native/v1-allure-dependency-cache-web-${{ runner.os }}-${{ github.sha }}-*' allure-results
+          for file in `ls allure-results/*tar`; do tar --use-compress-program='zstd --long=30' -xf $file; done
+          rm -f allure-results/*.tar
+          find . -maxdepth 1 -type d -name "allure-results-*" | while read dir; do
+          cp -r "$dir"/* allure-results
+          done
```

est-ce que la chose suivante serait équivalent ?

```suggestion
          cp allure-results-*/* allure-results
```

---

/usr/bin/cp: Argument list too long sur https://github.com/pass-culture/pass-culture-app-native/actions/runs/13694820621/job/38295484361?pr=7727
donc ça peut mener à des bugs